source/enumerate-collections.rst

```diff
@@ -362,8 +368,12 @@ Example return (in PHP, but abbreviated)::
       [5] => class MongoCollection#11 { }
     }
 
-Drivers MUST specify the ``nameOnly`` option when executing the ``listCollections`` command for this method,
-except drivers MUST NOT set ``nameOnly`` if a filter specifies any keys other than ``name``.
+Drivers SHOULD specify true for the ``nameOnly`` option when executing the
```

Comment:
SHOULD is still a change from MUST in the original text. I don't feel strongly about this, but if it wasn't intended you may want to revert.

Reply:
Ah I thought you wanted should based on the text of your other comment, I can change that for sure.

Reply:
Oh, I see how my last comment could have suggested that. I was just saying "should" in English and didn't mean to suggest spec language there. Apologies.

Reply:
No worries!